domain-models-runtime/src/main/java/org/folio/rest/persist/SQLConnection.java

```diff
@@ -7,8 +7,18 @@
 
   final PgConnection conn;
   final Transaction tx;
+  private Long timeId;
```

Comment:
Can you explain why this variable is not final? See https://github.com/folio-org/raml-module-builder/pull/657#discussion_r432584436 how to do this.

Reply:
I don't like idea to add additional redundant complexity of additional lambda-callbacks just for making this variable not final.

Reply:
You don't need to introduce the lambda to change `Long timeId` to `final Long timerId`. Only some reordering of lines, as shown in https://github.com/folio-org/raml-module-builder/pull/657#discussion_r432584436 , is needed.